app/services/session.js

```diff
@@ -16,6 +16,13 @@ export default Ember.Service.extend({
     return store.peekAll('user_profile').get('firstObject') || null;
   }).volatile(),
 
+  isOrderFulfilmentUser: Ember.computed('isLoggedIn', function() {
```

Comment:
Move this to model... `hasRole` property would be better choice in model. 
Session shouldn't be used to check if User has role or not